src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -178,13 +193,11 @@ def _write_pes_to_nwbfile(
     exclude_nans,
     pose_estimation_container_kwargs: Optional[dict] = None,
 ):
-    from ndx_pose import PoseEstimation, PoseEstimationSeries
```

Comment:
Moving this out of local scope, because this is an unexposed private submodule, might be OK...

Reply:
See last comment

We would prefer to keep this import scoped if possible, as a matter of practice